src/transactions/feed/TransactionFeedV2.tsx

```diff
@@ -370,14 +383,30 @@ export default function TransactionFeedV2() {
     [isFetching, data, standByTransactions]
   )
 
+  useEffect(
+    function hasLoadedAllTransactions() {
+      if (data && !data.pageInfo.hasNextPage && !data.pageInfo.endCursor) {
+        setAllTransactionsShown(true)
+      }
+    },
+    [data]
+  )
+
   useEffect(
     function handleError() {
       if (error === undefined) return
 
       Logger.warn(TAG, 'Error while fetching transactions', error)
-      dispatch(showToast(t('transactionFeed.error.fetchError'), null, null, null))
```

Comment:
showing a local toast here because we want a retry action, and we can't send (non serializable) callbacks as part of a redux action